web-ui/src/components/notes/Note.jsx

```diff
@@ -85,41 +85,43 @@ const Notes = (props) => {
     }
   }, [csrf, checkinId, currentUserId, pdlId]);
 
-  const handleNoteChange = (e) => {
+  const handleNoteChange = (content, delta, source, editor) => {
     if (Object.keys(note).length === 0 || !csrf) {
       return;
     }
-    const { value } = e.target;
+    
     setNote((note) => {
-      const newNote = { ...note, description: value };
+      const newNote = { ...note, description: content };
       updateNote(newNote, csrf);
       return newNote;
     });
-  };
+  }
 
   return (
       <Card className="note">
         <CardHeader avatar={<NotesIcon />} title={`Notes for ${currentMember?.name}`} titleTypographyProps={{variant: "h5", component: "h2"}} />
         <CardContent>
-          <div className="container">
```

Comment:
Just heads up, you'll probably have to update the snapshot tests for these files

Reply:
I'm glad you mentioned that.  I had run jest test and all tests passed but that is definitely not what should have happened. Looking into it now.